conf/webknossos.routes

```diff
@@ -176,8 +179,11 @@ GET           /api/projects/:name/pause
 GET           /api/projects/:name/resume                                        controllers.ProjectController.resume(name: String)
 GET           /api/projects/:name/usersWithOpenTasks                            controllers.ProjectController.usersWithOpenTasks(name:String)
 
+# GET           /api/statistics/webknossos                                        controllers.StatisticsController.oxalis(interval: String, start: Option[Long], end: Option[Long])
+# GET           /api/statistics/users                                             controllers.StatisticsController.users(interval: String, start: Option[Long], end: Option[Long], limit: Int)
+# GET           /api/statistics/assignments                                       controllers.StatisticsController.assignmentStatistics(start: Option[Long], end: Option[Long])
```

Comment:
why are statistics routes commented out? (might originate from react-router branch, which tom merged into this one)

Reply:
@hotzenklotz do you know something about this?